pytorch_lightning/plugins/training_type/ddp_spawn.py

```diff
@@ -214,6 +215,8 @@ def post_dispatch(self):
         best_path = self.mp_queue.get()
         last_path = self.mp_queue.get()
         self._results = self.mp_queue.get()
+        # TODO: verify to find corner cases or concurrency issues
+        self.lightning_module.trainer.logger_connector.spawn_extra_parameters.update(self.mp_queue.get())
```

Comment:
```suggestion
        self._results = self.mp_queue.get()
        extra_parameters = self.mp_queue.get()
        # TODO: verify to find corner cases or concurrency issues
        self.lightning_module.trainer.logger_connector.spawn_extra_parameters.update(extra_parameters)
```
for visual clarity, all get() from queue listed line by line :)